efel/_version.py

```diff
@@ -43,7 +53,7 @@ def get_config():
     cfg.style = "pep440-minor"
```

Comment:
Could you also change here to `"pep440"` please?

Reply:
I believe that file is generated by Versioneer based on the config. I'll run that again and probably squash the two commits into one.